test/cql-pytest/test_describe.py

```diff
@@ -736,6 +782,59 @@ def test_table_options_quoting(cql, test_keyspace):
 
 ### =========================== UTILITY FUNCTIONS =============================
 
+# Create random body of UDT using all UDTs from `udts`.
+# The functions tries to create complex field types,
+# by mixing lists, tuples, sets, etc...
+def create_udt_body(udts):
+    two_udts_type_probability = 0.3
+    nest_probability = 0.1
+
+    one_udt_type = lambda u: random.choice([
+        f"frozen<{u}>",
+        f"frozen<list<{u}>>",
+        f"list<frozen<{u}>>",
+        f"frozen<set<{u}>>",
+        f"set<frozen<{u}>>",
+        f"tuple<frozen<{u}>, int>",
+        f"frozen<tuple<{u}, int>>",
+        f"map<frozen<{u}>, int>",
+        f"map<int, frozen<{u}>>",
+        f"frozen<map<{u}, int>>",
+        f"frozen<map<int, {u}>>",
+    ])
+    two_udts_type = lambda u, w: random.choice([
+        f"frozen<tuple<{u}, {w}>>",
+        f"tuple<frozen<{u}>, frozen<{w}>>",
+        f"frozen<map<{u}, {w}>>",
+        f"map<frozen<{u}>, frozen<{w}>>"
+    ])
+    
+    fields = []
+    field_idx = 1
+    while udts:
+        # Select to create one_udt_type or two_udts_type
+        two_udts = False
+        if len(udts) > 1 and random.random() < two_udts_type_probability:
+            two_udts = True
+        
+        field_type = ""
```

Comment:
Nit: set it to `None`. This way, things will break in a more obvious way if you forget to initialize it in `if`/`else`.